dataactbroker/handlers/fileHandler.py

```diff
@@ -307,9 +310,10 @@ def checkSubmissionById(self, submission_id, file_type):
 
         """
         error = None
+        sess = GlobalDB.db().session
 
         try:
-            submission = self.interfaces.jobDb.getSubmissionById(submission_id)
+            submission = sess.query(Submission).filter_by(submission_id = submission_id).one()
         except ResponseException as exc:
```

Comment:
Yes, absolutely--we can't just ignore that logic. Just submitted a change for this. It's not _quite_ the same thing but should ensure that we're sending the same info back to the front-end.